content/docs/user-guide/project-structure/dvcyaml-files.md

```diff
@@ -87,22 +87,28 @@ $ dvc stage add -n a_stage "./a_script.sh > /dev/null 2>&1"
 $ dvc exp init './another_script.sh $MYENVVAR'
 ```
 
-### Parameter dependencies
+<admon type="tip">
+
+See also [Templating](#templating) (and **Dict Unpacking**) for useful ways to
+parametrize `cmd` strings.
```

Comment:
💅  Why not link directly to dict unpacking?

Reply:
Because some of the info about using params to template dvc.yaml is above the **Dict Unpacking** section.

Reply:
Or did you mean link both? I thought it would be repetitive, plus the latter is an H3 immediately below the one linked. But feel free to commit if you prefer:

```suggestion
See also [Templating](#templating) (and [Dict Unpacking]) for useful ways to
parametrize `cmd` strings.

[dict unpacking]:
  /doc/user-guide/project-structure/dvcyaml-files#dict-unpacking
```

The link ref should ideally be further down though...